MH4U Dex/CoreDataController.m

```diff
@@ -0,0 +1,446 @@
+//
+//  CoreDataController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/6/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "CoreDataController.h"
+#import <NSManagedObjectContext+Hydrate.h>
+#import "MonsterDrop.h"
+#import "AreaDrop.h"
+#import "Item.h"
+#import "Monster.h"
+#import "DamageZone.h"
+#import "Region.h"
+#import "Area.h"
+
+@interface CoreDataController ()
+
+@property (readwrite, strong, nonatomic) NSManagedObjectContext *managedObjectContext;
+@property (readwrite, strong, nonatomic) NSManagedObjectModel *managedObjectModel;
+@property (readwrite, strong, nonatomic) NSPersistentStoreCoordinator *persistentStoreCoordinator;
+
+@end
+
+@implementation CoreDataController
+
+NSString *const JSON = @"json";
+
+#pragma mark - Item Constants
```

Comment:
These `#pragma mark` statements might not be useful—it's worth checking the symbol navigator within the file to see how they display there and whether any of the constants you're declaring between `#pragma mark` statements show up there.